### ISSUE TITLE
Small issue when Downloading/Cloning

### ISSUE THREAD

**Issue by Montegro** [Tyrrrz/YoutubeExplode.Converter#23]
Not sure if this belongs here exactly since it's not an issue with the library from NuGet and more to do with Cloning/Downloading the project, but I've done so so I can update my old branch of this to work with the new YoutubeExplode project rewrite and uh, something's a bit off since all the dependencies seem to be missing according to Visual Studio. Would anyone happen to have a solution for this? Not sure if it's something about how this was packed to be uploaded or something I did on my end...
![image](https://user-images.githubusercontent.com/1057035/81613540-a92a4880-93ac-11ea-9916-5c902055a537.png)

**Comment by Tyrrrz** [comment]
Have you tried running `dotnet restore`?

**Comment by Montegro** [comment]
I just tried it out, this is what I got back 
![image](https://user-images.githubusercontent.com/1057035/81619956-b6e5cb00-93b8-11ea-8fc4-5c5a47048e57.png)

**Comment by Montegro** [comment]
Updated Visual Studio to the current new release version and that corrected that error, but I've got another... And that was related to the main directory for the project being different from what the solution expected. Corrected that and ran restore and it's all well and good. Thank you for your help. I'd've confirmed I found the solution sooner but dinner, dishes and all that. Thank for again for the help.
